deploy/cephcsi/image/Dockerfile

```diff
@@ -23,10 +23,10 @@ RUN source /build.env && \
 # test if the downloaded version of Golang works (different arch?)
 RUN ${GOROOT}/bin/go version && ${GOROOT}/bin/go env
 
-# FIXME: Ceph does not need Apache Arrow anymore, some container images may
-# still have the repository enabled. Disabling the repository can be removed in
-# the future, see https://github.com/ceph/ceph-container/pull/1990 .
-RUN dnf config-manager --disable apache-arrow-centos || true
+# TODO: remove the following cmd, when issue
+# https://github.com/ceph/ceph-container/issues/2034 is fixed.
```

Comment:
@Rakshith-R 
1. We don't need tcmu-runner packages in the ceph-csi containers! I would say we should keep it disabled forever.
2. Did you missed to diable `apache-arrow-centos` or is it intentionally removed?

Reply:
1. It shouldn't have been necessary to disable it in our build file, it should be handled by ceph containers team. This will be removed when that is resolved.
2. Its no longer necessary.